extensions/ql-vscode/src/databases/ui/db-panel.ts

```diff
@@ -57,6 +76,77 @@ export class DbPanel extends DisposableObject {
     await window.showTextDocument(document);
   }
 
+  // TODO: reconsider naming, since we're not really adding a database
+  private async addNewRemoteDatabase(): Promise<void> {
+    const quickPickItems = [
+      {
+        label: "$(repo) From a GitHub repository",
+        detail: "Add a remote database from a GitHub repository",
+        alwaysShow: true,
+        kind: "repo",
+      },
+      {
+        label: "$(organization) All repositories of a GitHub org or owner",
+        detail:
+          "Add a remote list of databases from a GitHub organization or owner",
+        alwaysShow: true,
+        kind: "owner",
+      },
```

Comment:
(not blocking) 
As we discussed offline, we might want to reconsider the wording around "adding databases". 🤔  But I'm also happy to get this in (since it matches the current design) and think about this a bit more later! ❤️